.travis.yml

```diff
@@ -2,7 +2,6 @@
 language: ruby
 
 rvm:
-  - 1.8.7
```

Comment:
Go ahead and leave this in so we can see the errors and I'll work to keep this supported.